src/helpers/attributes.js

```diff
@@ -7,6 +7,15 @@ export function hasSpread(attributes) {
   return attributes.some((attr) => attr.isJSXSpreadAttribute());
 }
 
+// Detects if the skip attribute is used
+export function hasSkip(attributes, { opts }) {
+  const skipAttribute = opts.skipAttribute || "__skip";
+  return attributes.some((attr) => {
+    const name = attr.get("name");
+    return name && name.node && (name.node.name === skipAttribute);
```

Comment:
I'm assuming the `name.node` check is to handle spread attributes? Could do `attr.isJSXAttribute() && attr.node.name.name === skipAttribute`